### PR TITLE
clock.s

### PR DESCRIPTION
**Pull request by PikalaxALT**

### CODE REVIEW

File: src/wallclock.c [hunk 1]
```diff
@@ -0,0 +1,1040 @@
...
+    return 1;
+}
+
+static u16 GetClockHandAngle(u16 a0, u8 command, u8 a2)
+{
+    u8 r1 = GetClockHandAngleChange(a2);
+    switch (command)
+    {
+        case 1:
+            if (a0) a0 -= r1;
```
[the remark this answers is not in the document]
Comment (camthesaxman) [reply src/wallclock.c]: case labels should be aligned with the opening brace.

File: src/clock.c [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+static void sub_809E858(struct Time *localTime)
+{
+    struct Time difference;
+    int minutes;
+
+    CalcTimeDifference(&difference, &gSaveBlock2Ptr->lastBerryTreeUpdate, localTime);
+    minutes = 24 * 60 * difference.days + 60 * difference.hours + difference.minutes;
+    if (minutes != 0)
+    {
+        if (minutes >= 0)
```
[the remark this answers is not in the document]
Comment (camthesaxman) [reply src/clock.c]: We tried this in Ruby, and it was the same way. I guess this is marginably more readable. https://github.com/pret/pokeruby/blob/master/src/engine/clock.c#L61

File: src/wallclock.c [hunk 3]
```diff
@@ -0,0 +1,1040 @@
...
+    }
+}
+
+static u8 GetClockHandAngleChange(u16 a0)
+{
+    if (a0 > 60)
+    {
+        return 6;
+    }
+    if (a0 > 30)
```
[the remark this answers is not in the document]
Comment (PikalaxALT) [reply src/wallclock.c]: My IDE complains loudly when I do that.

File: src/wallclock.c (hunk 1 above)
[the remark this answers is not in the document]
Comment (PikalaxALT) [reply src/wallclock.c]: I contest that convention as having the case labels indented is easier for me to read.

File: src/clock.c (hunk 2 above)
[the remark this answers is not in the document]
Comment (PikalaxALT) [reply src/clock.c]: @camthesaxman Done

**Pull request merged**